pyxem/signals/diffraction2d.py

```diff
@@ -949,19 +948,10 @@ def center_of_mass(
             this threshold value.
         mask : tuple (x, y, r), optional
```

Comment:
It should be mentioned in the docstring that the `mask` parameters are in pixel values. Ergo, not calibrated `axes_manager` units.

Reply:
Done!